src/repository.ts

```diff
@@ -842,46 +842,36 @@ export class Repository implements IDisposable {
         }
     }
 
-    async countIncomingAfterDelay(expectedDelta: number = 0, delayMillis: number = 3000): Promise<void> {
-        try {
```

Comment:
Why is `try`/`catch` no longer necessary here?

Reply:
All the `catch` does is re-throw the error.

Reply:
Haha. Makes sense 🤦‍♂️